packages/azure-http-specs/specs/azure/resource-manager/resources/mockapi.ts

```diff
@@ -791,7 +791,7 @@ Scenarios.Azure_ResourceManager_Resources_Singleton_update = passOnSuccess({
       },
     }),
     headers: {
-      "Content-Type": "application/merge-patch+json",
```

Comment:
PATCH in ARM is "application/json" (though part of it behavior like a json-merge-patch)
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/standbypool/resource-manager/Microsoft.StandbyPool/stable/2025-03-01/standbypool.json#L334

Reply:
We wrapped the body with `json`. Do we need to add the header once more?

Reply:
Emm, seems not. Other API does not have this `content-type` header.
Let me try remove it.